### PR TITLE
#161593693 Create Login page design

### PR DESCRIPTION
**Pull request by james947**
#### What does this PR do?
* Create login page design.
#### Description of Task to be completed?
* Allow users to gain access to the application.
#### How should this be manually tested?
* Run the application locally.
* Navigate to the login section.

#### What are the relevant pivotal tracker stories?
[#161593693](https://www.pivotaltracker.com/story/show/161593693)
#### Any background context you want to add?
* N/A
#### Screenshots
image

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
-Vue.config.productionTip = false
-Vue.use(VueAxios, {axios: axios})
+Vue.config.productionTip = false;
+Vue.use(VueAxios, axios);
+Vue.use(Vuetify, {
+  iconfont: 'mdi',
+});
 
+
+// Vue.axios.get('https://api-prod.andela.com', {
```
Comment (sayidu) [inline src/main.js]: Remove the commented out code.

File: src/components/Login/Login.vue [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  methods: {
+    login() {
+      window.location.href = 'https://api-prod.andela.com/login?redirect_url=http://localhost:8080'
+    }
+  }
+};
+
+</script>
+<style scoped>
+html {
```
[the remark this answers is not in the document]
Comment (james947) [reply src/components/Login/Login.vue]: @sayidu the PR is still under WIP, all the concerns will be addressed before the label is changed to review me.

File: src/assets/setup/_function.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+@function px-to-rem($size) {
```
Comment (kevinsamoei) [inline src/assets/setup/_function.scss]: Hey @james947 Let's rename this file `functions.scss` for consistency

File: src/assets/setup/_fonts.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
+$font_path: '~@/assets/fonts/';
+ @font-face {
+    font-family: DINPro-Light;
```
Comment (Nkemjiks) [inline src/assets/setup/_fonts.scss]: I believe the font used in the mockup is `Avenir`. Are we not supposed to stick to that?

Comment (james947) [reply src/assets/setup/_fonts.scss]: @Nkemjiks its DINPro according to the mockup, I needed to download another format for DINPro to make sure it looks exactly as what we have in the mockup.
image

Comment (Nkemjiks) [reply src/assets/setup/_fonts.scss]: I guess different fonts were used. This is my mockup

screen shot 2018-12-11 at 10 45 12 am

File: src/vuex/Store.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-restricted-syntax */
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+// import modules from './modules'
```
[the remark this answers is not in the document]
Comment (james947) [reply src/vuex/Store.js]: Thanks on it.

File: src/assets/setup/_fonts.scss (hunk 4 above)
Comment (james947) [reply src/assets/setup/_fonts.scss]: @Nkemjiks yes seems we using different fonts

Comment (Nkemjiks) [reply src/assets/setup/_fonts.scss]: I guess so. How best do we resolve this?

File: src/assets/setup/_function.scss (hunk 3 above)
Comment (james947) [reply src/assets/setup/_function.scss]: @kevinsamoei Feedback has been addressed

File: src/assets/setup/_fonts.scss (hunk 4 above)
Comment (james947) [reply src/assets/setup/_fonts.scss]: Lets take it to slack

File: src/vuex/Store.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (james947) [reply src/vuex/Store.js]: @faithngetich feedback addressed.

File: src/assets/components/login.scss [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+      }
+    }
+    @media only screen and (max-width: px-to-rem(823px)) and (orientation: landscape) {
+      #a-logo{
+        height: px-to-rem(48px);
+      }
+    }
+  }
+  
+
```
[the remark this answers is not in the document]
Comment (james947) [reply src/assets/components/login.scss]: @faithngetich all the concerns have been addressed, thanks for the feedback.

File: src/backend/axios/index.js [hunk 7]
```diff
@@ -1,10 +1,11 @@
-import axios from 'axios'
+import axios from 'axios';
```
Comment (g-kabu) [inline src/backend/axios/index.js]: Just curious, do we need this here since its a design implementation

File: src/components/Login/Login.vue [hunk 8]
```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="login-image">
+    <div class="french-rose">
+      <div class="square-top-left"></div>
+      <div class="square-top-right"></div>
+      <div class="square-bottom-left"></div>
+      <div class="square-bottom-right"></div>
+    </div> <!--end french-rose -->
+    <div class="homepage-text">
```
Comment (gr1d99) [inline src/components/Login/Login.vue]: good work @james947 , I just have a concern with class name `homepage-text`, this implementation is for `login` component therefore homepage seems not to be correct in my opinion

**Pull request merged**